src/main/java/com/minecolonies/coremod/generation/defaults/DefaultResearchProvider.java

```diff
@@ -612,6 +618,63 @@ public DefaultResearchProvider(final DataGenerator generator)
           .addItemCost(Items.DIAMOND, 1)
           .addEffect(ModBuildings.mysticalSite.getBuildingBlock(), 1)
           .addToList(r);
+
+        // Primary Research #6
+        final Research remembrance = new Research(new ResourceLocation(Constants.MOD_ID, "civilian/remembrance"), CIVIL)
+                .setTranslatedName("Remembrance")
+                .setSortOrder(6)
+                .setIcon(ModBlocks.blockHutGraveyard.asItem())
+                .addEffect(ModBuildings.graveyard.getBuildingBlock(), 1)
+                .addBuildingRequirement(ModBuildings.TOWNHALL_ID, 2)
+                .addItemCost(Items.BONE, 8)
+                .setTranslatedSubtitle("Our fallen shall not be forgotten!")
+                .addToList(r);
+
+        final Research resurectchance1 = new Research(new ResourceLocation(Constants.MOD_ID, "civilian/resurectchance1"), CIVIL)
+                .setParentResearch(remembrance)
+                .setTranslatedName("Resurect Change I")
+                .setTranslatedSubtitle("Use the right words")
+                .setSortOrder(1)
+                .setIcon(ModBlocks.blockHutGraveyard.asItem())
+                .addBuildingRequirement(ModBuildings.GRAVEYARD_ID, 3)
+                .addItemCost(Items.GHAST_TEAR, 1)
+                .addEffect(RESURECT_CHANCE, 1)
+                .addToList(r);
+
+        new Research(new ResourceLocation(Constants.MOD_ID, "civilian/resurectchance2"), CIVIL)
+                .setParentResearch(resurectchance1)
+                .setTranslatedName("Resurect Change II")
+                .setTranslatedSubtitle("Dance around and wave hands")
```

Comment:
see comments in lang file. Also, all places where you have `resurectchance` or `resurect` should be `resurrectchance`/`resurrect`